### PR TITLE
Updated profile description [WIP]

### PR DESCRIPTION
**Pull request by floWetzels**
Started refactoring the profile description, but still work-in-progress.

### CODE REVIEW

File: profile/arc_cwl_ro_crate.md [hunk 1]
```diff
@@ -6,31 +6,43 @@
...
+The ARC CWL profile mainly follows the [Workflow Run Crate](https://www.researchobject.org/workflow-run-crate/profiles/workflow_run_crate/) profile (which itself combines [Process Run Crate](https://www.researchobject.org/workflow-run-crate/profiles/process_run_crate/)  and [Workflow RO-Crate](https://about.workflowhub.eu/Workflow-RO-Crate/)) and extends it by providing means to annotate additional metadata and align terminology with other parts of an ARC.
 Computational workflows and laboratory workflows show many similarities, they typically only differ in how they are executed.
+In an ARC, the latter are described using the [ISA]() model, again seperating between a workflow description ([`LabProtocol`]()) and its execution ([`LabProcess`]()).
```
Comment (HLWeil) [inline profile/arc_cwl_ro_crate.md]: Link missing, did you meant to reference the ISA-RO-Crate profile for the ISA part, or the ISA abstract model?

**Review by HLWeil — COMMENTED** [review]
Looks great!

Just thinking about whether we should move the `Overview` part or a reduced version of it into the README, like we did in the [ARC profile](https://github.com/nfdi4plants/arc-ro-crate-profile)?

There you could also beautify it a bit by using the assay and run image from [here](https://arc-rdm.org/details/documentation-principle/) to showcase the similarity of assays and runs.

**Comment by floWetzels** [comment]
> Looks great!
> 
> Just thinking about whether we should move the `Overview` part or a reduced version of it into the README, like we did in the [ARC profile](https://github.com/nfdi4plants/arc-ro-crate-profile)?
> 
> There you could also beautify it a bit by using the assay and run image from [here](https://arc-rdm.org/details/documentation-principle/) to showcase the similarity of assays and runs.

Sounds like a good idea!

**Comment by floWetzels** [comment]
@HLWeil What do you think about the "Example Workflow Run Crate configuration in ARCs" paragraph (the text, not the figure)? I'm not sure what to do with it, since I also don't really understand everything explained there. Should we keep it, modifiy it or remove it?

File: profile/arc_cwl_ro_crate.md (hunk 1 above)
Comment (floWetzels) [reply profile/arc_cwl_ro_crate.md]: Fixed

**Comment by HLWeil** [comment]
> @HLWeil What do you think about the "Example Workflow Run Crate configuration in ARCs" paragraph (the text, not the figure)? I'm not sure what to do with it, since I also don't really understand everything explained there. Should we keep it, modifiy it or remove it?

I think the main point of this text is that to keep down complexity:
  - each `workflow` should have a `top-level workflow description` 
  - each `run` should have one `top-level run description`
  - the `top-level run description` should be the execution of one `top-level workflow description`

**Comment by HLWeil** [comment]
Very nice!

Could you move the images in the README into an `images` folder? Then we're goog to go.

Like in https://github.com/nfdi4plants/arc-ro-crate-profile

**Review by HLWeil — APPROVED** [review]

**Pull request merged**